File: MyCallBagEyeNeurology/app/src/main/java/com/opticpro/suite/ui/common/AppLogo.kt
// Copyright (c) Roberto Di Flumeri
package com.opticpro.suite.ui.common

import androidx.compose.foundation.Image
import androidx.compose.foundation.layout.size
import androidx.compose.foundation.shape.CircleShape
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.clip
import androidx.compose.ui.layout.ContentScale
import androidx.compose.ui.res.painterResource
import androidx.compose.ui.unit.Dp
import androidx.compose.ui.unit.dp
import com.opticpro.suite.R

/**
 * Logo ufficiale dell'app (occhio + rami neurali su sfondo nero), marchio Optools.
 */
@Composable
fun AppLogo(modifier: Modifier = Modifier, size: Dp = 48.dp) {
    Image(
        painter = painterResource(R.drawable.app_logo),
        contentDescription = null,
        contentScale = ContentScale.Crop,
        modifier = modifier.size(size).clip(CircleShape)
    )
}

File: MyCallBagEyeNeurologyIOS/Optools/Common/AppLogo.swift
// © Roberto Di Flumeri
import SwiftUI

/// Logo ufficiale dell'app (occhio + rami neurali su sfondo nero), marchio Optools.
struct AppLogo: View {
    var size: CGFloat = 48

    var body: some View {
        Image("app_logo")
            .resizable()
            .aspectRatio(contentMode: .fill)
            .frame(width: size, height: size)
            .clipShape(Circle())
    }
}
